### PR TITLE
Support intranet users

### PR DESCRIPTION
**Pull request by sbrunner**
Requires: https://github.com/camptocamp/c2cgeoportal/pull/5356

### CODE REVIEW

**Comment by sbrunner** [comment]
@fredj this is ready to be reviewed :-)

File: contribs/gmf/src/authentication/component.html [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<div ng-if="$ctrl.gmfUser.is_intranet">
+  <div class="form-group">
+    <span>{{'You are recognise as an intranet user.' | translate}}</span>
```
Comment (jwkaltz) [inline contribs/gmf/src/authentication/component.html]: You are recognized as an Intranet user.

Comment (sbrunner) [reply contribs/gmf/src/authentication/component.html]: Intranet don't take a capital letter (in opposition of Internet), see e.-g.:
https://en.wikipedia.org/wiki/Intranet

Comment (jwkaltz) [reply contribs/gmf/src/authentication/component.html]: OK... but you still need a "d" at the end of "recognize".

Comment (sbrunner) [reply contribs/gmf/src/authentication/component.html]: Already done ...

**Pull request merged**